pkg/cmd/server/start/start_master.go

```diff
@@ -558,7 +558,7 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunNamespaceController()
 		kc.RunPersistentVolumeClaimBinder(binderClient)
 		kc.RunPersistentVolumeProvisioner(provisionerClient)
-		kc.RunPersistentVolumeClaimRecycler(oc.ImageFor("recycler"), recyclerClient)
+		kc.RunPersistentVolumeClaimRecycler(oc.ImageFor("recycler"), recyclerClient, bootstrappolicy.InfraPersistentVolumeRecyclerControllerServiceAccountName, oc.Options.PolicyConfig.OpenShiftInfrastructureNamespace)
```

Comment:
I added that before fixing the namespace.  I'll test/verify it's not needed and then remove it.